packages/app/src/App.tsx

```diff
@@ -98,7 +101,10 @@ const AppRouter = app.getRouter();
 
 const routes = (
   <FlatRoutes>
-    <Navigate key="/" to="/catalog" />
+    <Navigate key="/" to="/home" />
+    <Route path="/home" element={<HomeIndexPage />}>
+      <HomePage />
+    </Route>
```

Comment:
I think it makes sense to have `/` render the homepage directly. WDYT?

Reply:
That would be my instinct as well. However, I guess if we consider it a kind of opt-in thing where people could chose to keep the Catalogue as their homepage, for instance, it feels very convenient for the App Integrator to just update the `to` attribute on the `<Navigate />` element.
But perhaps we should optimise for the user instead, and from their perspective `/` certainly makes more sense. 

Would love to hear other people's take on this as well.

Reply:
Yes, having adopters opt-in makes sense for newly created-apps.

However, if we are deciding to add this plugin in example-app, I think we should have it on `/`. `/` feels a bit more home like than `/home`. :) Ofcourse, our adopters are free to choose wherever they want this plugin to show up!

The redirect `/` to `/catalog` have always felt like a temporary hack to me, since we removed the ["welcome"](https://github.com/backstage/backstage/tree/master/plugins/welcome) plugin, and were waiting for a proper homepage plugin. It has caused minor issues like https://github.com/backstage/demo/issues/28

Reply:
Well put, big agree! Fixed in cfafa07. Interestingly, a bug prevented this from working, but thanks to the heroic efforts of @Rugvip coming out from vacation for a [fix](https://github.com/backstage/backstage/pull/6505), it now works as expected!

Reply:
We also run into issues that the active sidebar item doesn't work properly for `/` routes, is this still an issue?

Reply:
No, this was resolved by @Rugvip in [6505](https://github.com/backstage/backstage/pull/6505)! :tada:

Reply:
I presume the problem is that `/` matches every route, and is thus always active? I have resolved this in cca2bac. You need to pass an `end` flag (previous it was called `exact` I believe).

Reply:
The fix with `end` is pretty neat here, as the home route probably doesn't have sub routes (compared to the catalog). I tried it out locally, seems to work.

Reply:
That's great to hear! Not sure how we would solve it if sub routes indeed are introduced, but I'm sure people have run into this issue before, and there's probably a clever fix for it. From what I understand, NavLinks used to have a [`isActive` prop](https://reactrouter.com/web/api/NavLink), which allowed you to pass a function that would determine if the item was active or not. This doesn't seem to exist anymore.

Either way - let's deal with that when we get to it.

Reply:
Replying to the original comment about setting this to the default route - I'd be a bit put off if I was evaluating an infrastructure product and got random jokes with a smattering of layouts when I booted it up for the first time. This is in the example app rather than `create-app`, of course, but many people clone the repo to start playing around.

If we're to make it the default route, I think adding some more serious examples would be beneficial. Can we use Octokit to easily pull the 5 most recent open PRs from the Backstage core repo, for example? A few `backstage/master` build statuses? If we'd rather merge this PR and iterate, I'd suggest leaving it off the default route for now until we can work up better examples.

Reply:
That's a great comment, @timbonicus. If we cannot show anything remotely useful, it may just be confusing. However, we prefer to merge this without having to wait for more changes introducing components, so I think the better solution for now is to move the homepage away to `/home` and allow Catalog to continue being the start page for now, until we've introduced some components that can conceivably make out a decently useful homepage.
This was fixed in f1c7181